packages/experiments/src/components/UnifiedPicker/UnifiedPeoplePicker/examples/UnifiedPeoplePicker.Example.tsx

```diff
@@ -159,6 +159,7 @@ export const UnifiedPeoplePickerExample = (): JSX.Element => {
     noResultsFoundText: 'No suggestions',
     onFloatingSuggestionsDismiss: undefined,
     showSuggestionRemoveButton: true,
+    pickerWidth: '300px',
```

Comment:
looks like 300px is not the default then :) Sounds okay to me, just wanted to make sure you intended it to be this way.

Reply:
Yes, it was intended.  I went back and forth on this.... Making the default _on the floating suggestions_ 300px means that there's no need to change the UPP example, so that anyone who reuses it may miss that it's configurable.  I can change it if you like it better the other way.

Reply:
Yea, we can add a function in the UnifiedPicker, something similar like-
     targetElement: input.current?.inputElement,

so `const _renderFloatingPicker = () =>
    onRenderFloatingSuggestions({
      ...floatingSuggestionProps,
pickerWidth: props.floatingSuggestionProps.pickerWidth? props.floatingSuggestionProps.pickerWidth : '300px`

Reply:
Yes!